src/components/lib/VSeperator/VSeperator.vue

```diff
@@ -23,10 +23,10 @@ hr,
 
 hr {
   width: 100%;
-  border-top: 1px solid #000;
+  border-top: 1px solid #8d8d8d;
 }
 
 .vr {
-  border-left: 1px solid #000;
+  border-left: 1px solid #8d8d8d;
```

Comment:
Same as this https://github.com/arcana-network/developer-dashboard/pull/168/files#r1064300712